MessageCenter/Classes/Client/SendBirdClient.swift

```diff
@@ -122,9 +120,7 @@ public class SendBirdClient: ClientProtocol {
     }
     
     public var isConnected: Bool {
-        get {
-            return connected
-        }
+        return SBDMain.getConnectState().hashValue == 1 //Connection Opened
```

Comment:
```suggestion
         return SBDMain.getConnectState() == .open //Connection Opened
```
.**hashValue** will return an integer hash code of this value.